ecs-cli/modules/clients/aws/cloudformation/private_namespace_template.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//  http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package cloudformation
+
+func GetPrivateNamespaceTemplate() string {
+	return private_namespace_template
+}
+
+var private_namespace_template = `
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "AWS CloudFormation template to create a private dns namespace to enable ECS Service Discovery.",
+  "Parameters": {
+    "NamespaceDescription": {
+      "Type": "String",
+      "Description": "The description of the private dns namespace",
+      "Default": "Created by the Amazon ECS CLI"
+    },
+    "VPCID": {
```

Comment:
Nit: In the cluster template, we say `VpcId` -- not sure which is correct but perhaps worth being consistent?

Reply:
`VpcId` follows a Java variable name style, and `VPCID` follows the go style, I prefer the latter (though of course I suppose that since this is JSON we should follow JavaScript style, but I'm not sure what the is, do you?)